content/blog/35c3-refreshing-memories/contents+en.lr

```diff
@@ -2,7 +2,7 @@ author: L3D
 ---
 beitrag:
 
-In [the Wiki from 35c3](https://events.ccc.de/congress/2018/wiki/index.php/Static:Congress_Everywhere) are more Information about Congress Everywhere.
+In [the Wiki from 35c3](https://events.ccc.de/congress/2018/wiki/index.php/Static:Congress_Everywhere) there are more information about Congress Everywhere.
```

Comment:
"there is more information" wäre korrekt glaube ich

Reply:
stimmt :)

Reply:
Willst du hier noch anpassen?

Reply:
meh, base repo ist gelöscht.